### PR TITLE
Support email ask user to run in parallel

### PR DESCRIPTION
**Pull request by aviadl**
@gshriki specially for you

### CODE REVIEW

**Comment by omercnet** [comment]
spread the love!

File: Scripts/script-EmailAskUser.yml [hunk 1]
```diff
@@ -11,11 +11,20 @@ script: |-
...
 
   // Get entitlement
-  res = executeCommand('addEntitlement', {'persistent': args.persistent, 'replyEntriesTag': args.replyEntriesTag})
-  if (res[0].Type === entryTypes.error) {
-      return res;
+  var entitlement;
+  retries = args.retries || 10
+  for (i = 0 ; i < retries; i++) {
+      res = executeCommand('addEntitlement', {'persistent': args.persistent, 'replyEntriesTag': args.replyEntriesTag})
+      if (res[0].Type === entryTypes.error) {
```
Comment (gshriki) [inline Scripts/script-EmailAskUser.yml]: why not using `isError(res[0])`?

File: Scripts/script-EmailAskUser.yml [hunk 2]
```diff
@@ -138,4 +147,8 @@ args:
   - "true"
   - "false"
   defaultValue: "false"
+- name: retries
+  description: Indicates how many times to try and create an entitlement in case of failure
+  defaultValue: 10
 scripttarget: 0
+releaseNotes: Support script to run in parallel with itself
```
Comment (gshriki) [inline Scripts/script-EmailAskUser.yml]: Maybe rephrase - "Added support for parallel execution of the script, with better error handling"
